libs/clients/regulations-admin/src/lib/RegulationsAdminClientService.ts

```diff
@@ -0,0 +1,62 @@
+import { Inject } from '@nestjs/common'
+import { RESTDataSource, RequestOptions } from 'apollo-datasource-rest'
+import { DataSourceConfig } from 'apollo-datasource'
+import {
+  RegulationDraft,
+  ShippedSummary,
+  TaskListType,
+} from '@island.is/regulations/admin'
+import { RegulationsAdminClientConfig } from './RegulationsAdminClientConfig'
+import { ConfigType } from '@nestjs/config'
+
+export class RegulationsAdminClientService extends RESTDataSource {
```

Comment:
Would be nice to refactor this to use OpenAPI and generated client with Enhanced Fetch.